### ISSUE TITLE
ucx cm test failure

### ISSUE THREAD

**Issue by paklui** [openucx/ucx#2735]
observed some failures in UCX gtest on the HPCAC hali systems with MLNX_OFED_LINUX-4.4-1.0.0.0
According to Yossi, these will be fixed in upcoming MLNX_OFED

```
contrib/../configure --enable-optimizations --disable-logging --disable-debug --disable-assertions --disable-params-che
ck | tee -a $LOG
make -C test/gtest test | tee -a $LOG

[  FAILED  ] cm/uct_flush_test.am_flush_ep_no_comp/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/uct_flush_test.am_flush_ep_no_comp/1, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/uct_flush_test.am_flush_iface_no_comp/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/uct_flush_test.am_flush_iface_no_comp/1, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/uct_flush_test.am_flush_ep_nb/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/uct_flush_test.am_flush_ep_nb/1, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/uct_flush_test.am_pending_flush_nb/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/uct_flush_test.am_pending_flush_nb/1, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy/1, where GetParam() = cm/mlx5_0:1/loopback
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy/2, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy/3, where GetParam() = cm/mlx5_2:1/loopback
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy_keep_data/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy_keep_data/1, where GetParam() = cm/mlx5_0:1/loopback
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy_keep_data/2, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/uct_p2p_am_test.am_bcopy_keep_data/3, where GetParam() = cm/mlx5_2:1/loopback
[  FAILED  ] cm/test_uct_pending.pending_async/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/test_uct_pending.pending_async/1, where GetParam() = cm/mlx5_2:1
[  FAILED  ] cm/test_uct_pending.pending_ucs_ok_dc_arbiter_bug/0, where GetParam() = cm/mlx5_0:1
[  FAILED  ] cm/test_uct_pending.pending_ucs_ok_dc_arbiter_bug/1, where GetParam() = cm/mlx5_2:1

```

**Comment by yosefe** [comment]
Fix: http://l-gerrit.mtl.labs.mlnx:8080/#/c/upstream/linux/+/171670/ (mellanox internal link)

**Comment by yosefe** [comment]
fixed in MLNX_OFED-4.4-2.0.7.0
